### PR TITLE
Hide second iterm relax tooltip

### PR DESCRIPTION
**Pull request by McGiverGim**
Fixes https://github.com/betaflight/betaflight-configurator/issues/2719

image

This PR includes two commits:
- The first, hides the cutoff iterm relax tooltip when the iterm relax is disabled.
- The second one, gives a little more room to the first column of the table. This is a workaround to the real problem: we can have some translations bigger than the original, so the column must auto adjust. If someone wants to look at the auto adjust and fix it in a real manner will be great. But for now, with this change the current translations will fit. 

The final result, in native english is:
image

### CODE REVIEW

**Comment by Asizon** [comment]
Looks good @McGiverGim, I forget this needed fix sorry.

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by ctzsnooze — APPROVED** [review]

**Comment by ctzsnooze** [comment]
Perhaps for long names we can include a hyphen?

**Comment by haslinghuis** [comment]
CamelCase preferred

**Pull request merged**
